spec/bike_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rspec'
+require_relative '../lib/dependencies'
+
+RSpec.describe Bike do 
+  before { DeliveryService.new(1,1) }
```

Comment:
why do we need it?

Reply:
it was necessary because the park was inside initialize , now it fixed and removed